crates/red_knot_python_semantic/resources/mdtest/scopes/moduletype_attrs.md

```diff
@@ -9,7 +9,7 @@ is unbound.
 ```py
 reveal_type(__name__)  # revealed: str
 reveal_type(__file__)  # revealed: str | None
-reveal_type(__loader__)  # revealed: LoaderProtocol | None
+reveal_type(__loader__)  # revealed: @Todo(instance attribute on class with dynamic base)
```

Comment:
When we now look up `types.ModuleType.__loader__`, we try to access the `__get__` member on `LoaderProtocol | None`, which triggers this `@Todo`. I thought this wasn't important enough to fix this, but let me know if you think otherwise.